Website/Packages/datax-query/src/modules/query/queryModels.js

```diff
@@ -0,0 +1,26 @@
+// *********************************************************************
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License
+// *********************************************************************
+
+export function getDefaultQuery(enableLocalOneBox) {
+    if (enableLocalOneBox) {
+        return defaultQueryLocal;
+    } else {
+        return defaultQuery;
+    }
+}
+//User can use the sample or tutorial or intellisense to have starting query. Below allows default to have 5 blank lines.
+export const defaultQuery = `
+
+`;
+
+export const defaultQueryLocal = `--DataXQuery--
+events = SELECT MAX(temperature) as maxTemp

```

Comment:
Is this used anywhere?  I thought we had a different default.

Reply:
Yes.
Used by the below function:
export function getDefaultQuery(enableLocalOneBox) {... line 6 in queryModels.js
... which is used in the local onebox experience.